lib/ansible/plugins/lookup/consul_kv.py

```diff
@@ -61,7 +61,12 @@
 
 import os
 import sys
-from urlparse import urlparse
+
+try:
+    from urllib.parse import urlparse  # python3
+except ImportError:
+    from urlparse import urlparse      # python2
+
```

Comment:
Instead of the try/except dance, `urlparse` should be imported from `ansible.module_utils.six` instead.  Such as:

```
import ansible.module_utils.six.moves.urllib.parse as urlparse
```

or

```
from ansible.module_utils.six.moves.urllib.parse import urlparse
```